src/plugins/__mocks__/database.ts

```diff
@@ -1,6 +1,8 @@
-import { FastifyPluginAsync } from 'fastify';
 import { createMockPool, createMockQueryResult } from 'slonik';
-import { Database, DatabasePluginOptions } from '../database';
+
+import { FastifyPluginAsync } from 'fastify';
+import {Database} from '@graasp/sdk';
```

Comment:
This seems weirdly formatted. Could it be that since the `yarn pre-commit` hook is commented out, this was not caught by the linter?